GNSSLogger/app/src/main/java/com/google/android/apps/location/gps/gnsslogger/MainActivity.java

```diff
@@ -141,12 +140,12 @@ protected void onCreate(Bundle savedInstanceState) {
 
   protected PendingIntent createActivityDetectionPendingIntent() {
     Intent intent = new Intent(this, DetectedActivitiesIntentReceiver.class);
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
-      return PendingIntent.getBroadcast(this, 0, intent,
-              PendingIntent.FLAG_UPDATE_CURRENT | PendingIntent.FLAG_MUTABLE);
-    } else {
-      return PendingIntent.getBroadcast(this, 0, intent, PendingIntent.FLAG_UPDATE_CURRENT);
-    }
+    return PendingIntentCompat.getBroadcast(
+        /* context= */ this,
```

Comment:
> Could you please remove the comments indicating the parameter names? Android Studio adds the parameter name hints so if the comments are included as well it duplicates this information visually.
> 
> ![image](https://user-images.githubusercontent.com/928045/181064117-df81e140-2b9a-4832-bdc6-8b7367fdc436.png)

Thanks, I've added this because some people don't use IDE. Also, parameter hint is kind of Google standard format. Maybe you can confirm it and then we could reconsider whether to add it or not.